otx/cli/utils/hpo.py

```diff
@@ -232,23 +230,28 @@ def run_hpo_trainer(
     HpoManager.set_hyperparameter(hyper_parameters, hp_config["params"])
     print(f"hyper parameter of current trial : {hp_config['params']}")
 
-    dataset_class = get_dataset_class(task_type)
     # TODO remove None from dataset_paths
-    dataset = dataset_class(
+    data_roots = dict(
         train_subset={  # type: ignore
-            "ann_file": dataset_paths.get("train_ann_file", None),
             "data_root": dataset_paths.get("train_data_root", None),
         },
         val_subset={  # type: ignore
-            "ann_file": dataset_paths.get("val_ann_file", None),
             "data_root": dataset_paths.get("val_data_root", None),
         },
     )
 
+    # Datumaro
+    datumaro_adapter = get_dataset_adapter(task_type)
```

Comment:
I just found that datumaro_adapter is initially make datumaro dataset in initialize function as they called. Then isn't it okay to remove import_dataset like cli/tools/train.py?

Reply:
I missed this part. As you said, `_import_dataset` will not be used anymore.